### PR TITLE
[Proposal] - Solidify getting started with Coursier

### PR DESCRIPTION
**Pull request by ckipp01**
👋🏼 Hi all. Here is a proposal to solidify the "Getting Started with Scala" via Coursier by myself and @tpolecat.

We're looking forward to discussing this!

### CODE REVIEW

File: proposals/026-solidify-getting-started-with-coursier.md [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+
+- While this could be an entire proposal on its own, there are multiple
+    sbt launchers that are floating around the ecosystem. When using Coursier
+    it's not fully clear which is actually being installed and users at times
+    are met with very slight differences between the launchers that are
+    extremely difficult for newcomers and at times veterans to spot, unless the
+    user is knowledgeable about the slight differences between launchers. The
+    [sbt documentation](https://www.scala-sbt.org/download.html) makes no
+    mention of Coursier being an official way to download the _official_
+    launcher.
```
Comment (eed3si9n) [inline proposals/026-solidify-getting-started-with-coursier.md:240]: +1. Unless it's byte-for-byte identical Bash script (runner) and JAR file (sbt-launch.jar), I really wish all third-party project sbt runners to stop using the _name_ sbt to identify themselves either on the command line or on the documentation so the users know what they are getting themselves into.

File: proposals/026-solidify-getting-started-with-coursier.md [hunk 2]
```diff
@@ -0,0 +1,285 @@
...
+`cs setup`).
+
+In most cases simply downloading Scala is not what users need to do. We suggest
+that these pages be de-emphasized in favor of Getting Started.
+
+Related to this, some instructions do mention installing the `scala3-repl` via
+Coursier, and running a file with it, but the `scala3-repl` installed by
+Coursier cannot [serve as a
+runner](https://github.com/lampepfl/dotty/issues/12811). This should be
+remedied.
```
Comment (julienrf) [inline proposals/026-solidify-getting-started-with-coursier.md:66]: I suggest also mentioning https://github.com/scala/scala-lang/pull/1209, which provides some elements of discussion, to give a broader overview of the situation.

Comment (ckipp01) [reply proposals/026-solidify-getting-started-with-coursier.md:66]: The linked conversation would be more appropriate down below when we discuss the unofficial sbt being installed, not here where we are focusing on being unable to run a scala file with `scala3-repl`, correct?

Comment (ckipp01) [reply proposals/026-solidify-getting-started-with-coursier.md:66]: I ended up referencing this down below.

**Review by sjrd — APPROVED** [review]

**Pull request merged**